lucene/core/src/test/org/apache/lucene/util/hnsw/KnnGraphTester.java

```diff
@@ -132,13 +135,13 @@ private void run(String... args) throws Exception {
           if (iarg == args.length - 1) {
             throw new IllegalArgumentException("-beamWidthIndex requires a following number");
           }
-          HnswGraphBuilder.DEFAULT_BEAM_WIDTH = Integer.parseInt(args[++iarg]);
```

Comment:
With this change, we no longer have any need to make these static variables writable - let's change them to `final` in `HnswGraphBuilder`

Reply:
Made them final and also made changes to `TestKnnGraph.java` to accommodate this change.